mappings/net/minecraft/text/Text.mapping

```diff
@@ -1,9 +1,83 @@
 CLASS net/minecraft/class_2561 net/minecraft/text/Text
+	COMMENT A json text.
+	COMMENT
+	COMMENT <p>Each json text has a tree strucutre, embodying all its {@link
+	COMMENT #getSiblings() siblings}. To iterate contents in the text and all
+	COMMENT its siblings, call {@code visit} methods.</p>
+	COMMENT
+	COMMENT <p>This interface does not expose mutation operations. For mutation,
+	COMMENT refer to {@link MutableText}.</p>
+	COMMENT
+	COMMENT @see MutableText
```

Comment:
Not needed because it's linked right above.

Reply:
see is more outstanding.